src/NexusMods.App.UI/LeftMenu/Downloads/DownloadsLeftMenuViewModel.cs

```diff
@@ -62,6 +66,7 @@ public DownloadsLeftMenuViewModel(
         {
             gameRegistry.InstalledGames
                 .ToObservableChangeSet()
+                .FilterOnObservable(gameInstallation => IsManagedGameObservable(gameInstallation, _connection))
```

Comment:
Instead of doing this we can directly observe the loadouts and then group by game installation